File: frontend/js/feedback.js
const API =
    "http://localhost:8080/api/feedback";

async function loadFeedback(){

    const response = await fetch(API);

    const data = await response.json();

    const table =
        document.getElementById("feedbackTable");

    table.innerHTML = `

        <tr>
            <th>Candidate</th>
            <th>Communication</th>
            <th>Technical</th>
            <th>Problem Solving</th>
            <th>Recommendation</th>
            <th>Comments</th>
        </tr>
    `;

    data.forEach(feedback => {

        table.innerHTML += `

            <tr>
                <td>${feedback.candidate}</td>
                <td>${feedback.communication}</td>
                <td>${feedback.technical}</td>
                <td>${feedback.problemSolving}</td>
                <td>${feedback.recommendation}</td>
                <td>${feedback.comments}</td>
            </tr>
        `;
    });
}

loadFeedback();

File: backend/src/main/java/com/paltech/interviewplatform/model/Feedback.java
package com.paltech.interviewplatform.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "feedback")

@Data
@NoArgsConstructor
@AllArgsConstructor

public class Feedback {

    @Id
    private String id;

    private String candidate;

    private int communication;

    private int technical;

    private int problemSolving;

    private String recommendation;

    private String comments;
}